flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/source/datagen/SequenceGenerator.java

```diff
@@ -53,6 +54,8 @@
      * @param end End of the range of numbers to emit.
      */
     public SequenceGenerator(long start, long end) {
+        Preconditions.checkArgument(
```

Comment:
The requirements of this class should be reflected in the JavaDoc as well.

Reply:
Done.